docs/DeveloperGuide.md

```diff
@@ -64,6 +64,25 @@ and extract out the necessary information if there is a match.
 This will be used by each command's respective parsing method and returns the relevant parsed `Command`
 object to `SuperTracker+handleCommands()`
 
+### New Command
+The following is a class diagram of the NewCommand and its relevant dependencies
+![NewCommandClass](uml-diagrams/NewCommandClass.png)
+The `NewCommand` class implements the `Command` interface and is responsible for handling the creation of new items in the `Inventory`.
+A NewCommand instance is created by the `ParseNewCommand` method called by Parser, which ensures that the provided parameters (name, quantity, price) are valid.
```

Comment:
It appears theres a small typo in the capitalization of the method name here. Could you fix this so that the method name is consistent with that in our code?
```suggestion
A NewCommand instance is created by the `parseNewCommand` method called by Parser, which ensures that the provided parameters (name, quantity, price) are valid.
```